content/concepts/loaders.md

```diff
@@ -1,6 +1,163 @@
 ---
 title: Loaders
 sort: 2
+contributors:
+  - manekinekko
 ---
-> module.loaders configuration option
-> test, include, exclude, loader, query
+
+## What are loaders?
+
+Loaders are transformations that are applied on a resource file of your app. They are functions (running in node.js) that take the source of a resource file as the parameter and return the new source.
```

Comment:
* /s/app/application
* /node.js/Node.js

This might be a good place to mention something about the default evaluation order (right to left, bottom to up) though it can be later too.